transport/graphsync/graphsync.go

```diff
@@ -980,6 +1037,11 @@ func (c *dtChannel) pause() error {
 	c.lk.Lock()
 	defer c.lk.Unlock()
 
+	if c.gsKey == nil {
```

Comment:
So, this is to support the case where `PauseChannel` is called before we've started the graphsync transfer for a channel that's already opened, right ?